_includes/nav.html

```diff
@@ -1,5 +1,5 @@
   <a href="https://thegooddocsproject.dev/" style="color:#ffffff;">Home</a> |
-  <a href="https://thegooddocsproject.dev/about.html" style="color:#ffffff;">About</a> | 
-  <a href="https://thegooddocsproject.dev/licences.html" style="color:#ffffff;">Licences</a> | 
-  <a href="https://thegooddocsproject.dev/contribute.html" style="color:#ffffff;">Contribute</a> | 
-  <a href="https://thegooddocsproject.dev/contact.html" style="color:#ffffff;">Contact Us</a> | 
\ No newline at end of file
+  <a href="https://thegooddocsproject.dev/licences.html" style="color:#ffffff;">Licences</a>
```

Comment:
Can we make this hyperlinks relative (licences.html) instead of absolute (https://thegooddocsproject.dev/licences.html)?
Doing so will mean we can test the site in a staging environment, or local environment.
Apply the same comment to other links too.

Reply:
Done.